### PR TITLE
Added new "Add Attribute" command and new "Create MISP Event" command

### PR DESCRIPTION
**Pull request by dem-evgeny**
Added new "Add Attribute" command and new "Create MISP Event" command

### CODE REVIEW

File: Integrations/integration-MISP.yml [hunk 1]
```diff
@@ -417,6 +423,92 @@ script:
...
+
+
+    var getNowTime = function() {
+        var time = new Date();
+        time = addMinutes(time,time.getTimezoneOffset());
+        var month = "0" + (time.getMonth()+1);
+        var day = "0" + time.getDate();
+        var hours = "0" + time.getHours();
+        var minutes = "0" + time.getMinutes();
+        var seconds = "0" + time.getSeconds();
```
Comment (yuvalbenshalom) [inline Integrations/integration-MISP.yml]: what is the reason for this? looks like you assign value day /hours/ minutes/second but never use them.

File: Integrations/integration-MISP.yml [hunk 2]
```diff
@@ -417,6 +423,92 @@ script:
...
+
+        var body = { 'Event': {
+            'date': getNowTime(),
+            'threat_level_id': '1',
+            'published': args.published,
+            'info': args.info,
+            'analysis': '0',
+            'distribution': '0',
+            'Attribute': [currentArgs] } };
+
```
Comment (yuvalbenshalom) [inline Integrations/integration-MISP.yml]: are you sure about all of those constants?  
'threat_level_id': '1'
 'analysis': '0'
 'distribution': '0'

maybe some of them needs to be arguments to the command?

File: Integrations/integration-MISP.yml (hunk 1 above)
Comment (dem-evgeny) [reply Integrations/integration-MISP.yml]: I do use it here:
var body = { 'Event': {
            'date': getNowTime(),
            'threat_level_id': '1',
            'published': args.published,
            'info': args.info,
            'analysis': '0',
            'distribution': '0',
            'Attribute': [currentArgs] } };

File: Integrations/integration-MISP.yml (hunk 2 above)
Comment (dem-evgeny) [reply Integrations/integration-MISP.yml]: Yes, I checked it at the customer

Comment (dem-evgeny) [reply Integrations/integration-MISP.yml]: Fixed

File: Integrations/integration-MISP.yml (hunk 1 above)
Comment (dem-evgeny) [reply Integrations/integration-MISP.yml]: Fixed

**Pull request merged**